uui/components/inputs/Slider.module.scss

```diff
@@ -1,5 +1,14 @@
 .root {
     --uui-slider-tooltip-bg: var(--uui-color-70);
+    --uui-slider-tooltip-bg-hover: var(--uui-primary-20);
+    --uui-slider-line-bg: var(--uui-neutral-30);
+    --uui-slider-line-filled-bg: var(--uui-primary-50);
+    --uui-slider-line-active-bg: var(--uui-primary-50);
+    --uui-slider-dot-bg: var(--uui-surface-main);
+    --uui-slider-dot-bg-hover: var(--uui-primary-20);
+    --uui-slider-handle-bg: var(--uui-primary-50);
+    --uui-slider-shadow: var(--uui-shadow-level-1);
+    --uui-slider-scale-text-color: var(--uui-text-primary);
```

Comment:
rename to '--uui-slider-scale-text', we don't add 'color' prefix for our variables